src/elementary/test/fromEndfReactionNumber.test.hpp

```diff
@@ -21,6 +21,8 @@ SCENARIO( "fromEndfReactionNumber" ) {
       CHECK( ReactionID( "n,Fe56->n,n,he4,Cr51" ) == fromEndfReactionNumber( neutron, Fe56_e0, 24 ) );
 
       CHECK( ReactionID( "n,Fe56->n,n,n,n,Fe53" ) == fromEndfReactionNumber( neutron, Fe56_e0, 37 ) );
+      CHECK( ReactionID( "n,Fe56->disappearance" ) == fromEndfReactionNumber( neutron, Fe56_e0, 101 ) );
+      CHECK( ReactionID( "n,Fe56->absorption" ) == fromEndfReactionNumber( neutron, Fe56_e0, 27 ) );

```

Comment:
Since you're adding them to this test, add them to the toEndfReactionNumber test as well. Move them up with the non-elastic and anything one.

You should also duplicate them for the protons in both tests. That way the tests will be in sync. I did forget to add the non-elastic and anything ones for the proton though.